includes/blocks/class-sensei-course-outline-module-block.php

```diff
@@ -86,9 +86,9 @@ public function render_module_block( $block, $course_id, $outline_attributes ) {
 					<h2 class="wp-block-sensei-lms-course-outline-module__title">' . $title . '</h2>
 					' . $progress_indicator .
 			( ! empty( $outline_attributes['collapsibleModules'] ) ?
-				'<button type="button" class="wp-block-sensei-lms-course-outline__arrow sensei-collapsible__toggle">
+				'<button type="button" class="wp-block-sensei-lms-course-outline__arrow sensei-collapsible__toggle" aria-expanded="true">
 						' . Sensei()->assets->get_icon( 'chevron-up' ) . '
-						<span class="screen-reader-text">' . esc_html__( 'Toggle module content', 'sensei-lms' ) . '</span>
+						<span class="screen-reader-text">' . $title . '</span>
```

Comment:
Better to tell users what they are expanding or collapsing. Toggle is implied since the button is in play.